### ISSUE TITLE
在确定需求后是否真的需要完全重新开发？

### ISSUE THREAD

**Issue by andriyhuang** [Rominez/league-of-Chinese-programmers#2]
从时间成本角度的考虑我不太建议来重新开发，最好是能以快速支撑业务目标为目的，而且大家也不要陷入到技术选型之中，我们不是为了做一个大家都喜欢技术系统而是需要一个安全的平台来做到信息的公平公开，所以在最终的实施方式上还值得好好考量！

**Comment by Rominez** [comment]
在理。我的初步想法也只是能让大家对公司，像买家对商品那样进行点评，用户进行点评时要有自己的立场角色，公司要有总体的评价统计，这是基本核心。其他功能慢慢扩大

**Comment by sylviatanggit** [comment]
直接参考看准网或类似的，我建议做简单一点，分内部评价和外部人员的灌水。

**Comment by MarcSteven** [comment]
It's not a great idea because the information is so important ,how to avoid this case ?Just monitor the activities of enterprise and then let it display in public ,that's why you want to do ...If do as taboo or other websites ,it always bring more advertisements ...
Just one purpose:

> Avoid the unfair work time  ,avoid 996 ,so simple.

**Comment by Rominez** [comment]
I had thinked this. Now we have a website names KanZhun that do something same, and I had made a test. I fined they had do nothing about avoid this question, any people could choose if he had worked in an enterprise, and then say anything whatever good or bad as an insider. Actually, it do helped people, but I really dislike that. 
I try to build someplace that peoples can share their situation in the company, and should not care about be fire; someplace that peoples can get infomation about is this job he can accept. Of course I want build a system, and people took up legal weapon fight with their bad boss, but its too hard in China. Before we fight, we also need this job. I think less Chinese will really choose to charge their company becouse work overtime, and no one will help in substance to make this fight, at least for now.
I also heard that many company already legalized their work overtime system by 特殊工时制行政审批( sorry, I really not good at English, QwQ ).

**Comment by torchina** [comment]
**聊天室 - 没有自我审查**

**https://gitter.im/torchina/community**

**Tor 浏览器 8.0.8 - 绕过gfw**

![Tor Browser](https://github.com/torchina/get/blob/master/2019-04-06_22-27-39.png?raw=true)

**https://torchina.github.io/get**

**Comment by MarcSteven** [comment]
> I had thinked this. Now we have a website names KanZhun that do something same, and I had made a test. I fined they had do nothing about avoid this question, any people could choose if he had worked in an enterprise, and then say anything whatever good or bad as an insider. Actually, it do helped people, but I really dislike that.
> I try to build someplace that peoples can share their situation in the company, and should not care about be fire; someplace that peoples can get infomation about is this job he can accept. Of course I want build a system, and people took up legal weapon fight with their bad boss, but its too hard in China. Before we fight, we also need this job. I think less Chinese will really choose to charge their company becouse work overtime, and no one will help in substance to make this fight, at least for now.
> I also heard that many company already legalized their work overtime system by 特殊工时制行政审批( sorry, I really not good at English, QwQ ).

Yeah  right ,I think we should do something let the company follow these rules ,but it's not easy ,because someone needs money to make a living ,so that's why the issue is still existing ,if only one people ,nobody cares about it ,but there are lots of people fighting for it ,it's possible to make the rule fair ,for instance the same role has fair salary.Based on the work experiences adjust the salary ...
